src/pages/Home/components/CardsMenu/index.tsx

```diff
@@ -44,34 +46,77 @@ const CardLink = styled(Link)`
 
 const StyledCard = styled(Card)`
   width: ${CARD_WIDTH_PX}px;
-  aspect-ratio: 3 / 2;
 `
 
 const StyledCardMedia = styled(CardMedia)`
-  height: 100%;
+  aspect-ratio: 3 / 2;
+`
+
+const StyledImage = styled(Image)`
+  transition: transform 500ms ease-out;
 `
 
 const StyledCardContent = styled(CardContent)`
+  position: relative;
+  padding: 1em;
   min-height: 0;
 `
 
+const StretchedDiv = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
 const StyledHeading = styled(Heading)`
   display: flex;
-  gap: 0.75rem;
   margin: 0;
+  line-height: 1;
+  gap: 0.75rem;
 
   & > span {
-    margin-top: 2px;
+    margin-top: -2px;
   }
 `
 
-const StyledImage = styled(Image)`
-  transition: transform 500ms ease-out;
+const PaddedIcon = styled(Icon)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 28px;
+  height: 18px;
+`
+
+const PopupContainer = styled.div`
+  display: flex;
+  position: absolute;
+  top: -${CARD_WIDTH_PX / 1.5}px;
+  left: 0;
+  align-items: flex-end;
+  transition: top 0.25s ease-out, height 0.25s ease-out;
+  width: 100%;
+  height: ${CARD_WIDTH_PX / 1.5}px;
+  overflow: hidden;
+`
+
+const PopupContent = styled.div`
+  background: ${themeColor('tint', 'level2')};
+  padding: 1em 1em 0;
+  width: 100%;
+  color: black;
+`
+
+const PopupParagraph = styled(Paragraph)`
+  line-height: 1.25;
+  font-size: 0.75em;
 `
 
 function CardsMenu() {
   const { t } = useTranslation()
 
+  const [showDesktopVariant] = useMatchMedia({ minBreakpoint: 'laptop' })
```

Comment:
Waarom wordt hier een verschil in gemaakt, desktop of niet? Alleen om het 'i'-icoontje wel of niet te tonen? Maar de tegel blifjt altijd even groot, ongeacht de breedte van je scherm. Waarom zou je dan soms wel en soms niet het 'i'-icoontje laten zien?

Reply:
Hi Ian, I'm responding in English: I agree that it's maybe not a great solution. It was decided with the stake-holders that we do not show the 'i' on smaller screens because there is no touch functionality. Users will usually not get to see these descriptions, which was deemed fine, as they are not essential for understanding what a tile does.